lib/rules/at-rule-semicolon-space-before/README.md

```diff
@@ -0,0 +1,45 @@
+# at-rule-semicolon-space-before
+
+Require a single space or disallow whitespace before the semicolons of regular (non-nested) at-rules.
```

Comment:
> I was trying to talk about at-rules like this one

Oh I see where you're coming from. 

In the `@font-face` case those semi-colons [belong to](https://www.w3.org/TR/css-fonts-3/#font-face-rule) the "block of descriptor declarations", and not to the at-rule itself. In stylelint we generally generalise blocks like these as "declaration blocks".

Having said that...

> Should I amend this to say "at-rules without declaration blocks" or some such?

... I don't think there's much need to be specific in the rule description as the examples quite clearly illustrate we're talking about semicolons after the likes of `@import` and `@charset`. I think "Require a single space or disallow whitespace after the semicolons of at-rules." will suffice here.